### PR TITLE
Updates to Jenkinsfile, dockerfile, build template to run in Openshift/Jenkins

### PR DESCRIPTION
**Pull request by accorvin**
These are the necessary changes to run cvengine on our production Jenkins/Openshift instance with passing in container validation parameters as job input parameters.

### CODE REVIEW

**Comment by accorvin** [comment]
You can see a successful run of this in our Jenkins instance here: https://jenkins-continuous-infra.apps.ci.centos.org/blue/organizations/jenkins/container-validation/detail/container-validation/2/pipeline

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
-fedmsgRetryCount = 120
-
 // Execution ID for this run of the pipeline
 executionID = UUID.randomUUID().toString()
 
 // Pod name to use
-podName = 'fedora-atomic-' + executionID + '-' + TARGET_BRANCH
+podName = 'cvengine-' + executionID
 
 library identifier: "ci-pipeline@${env.ghprbActualCommit}",
```
Comment (johnbieren) [inline ci/Jenkinsfile]: I'm not sure you want the $commit version of the ci-pipeline repo, you can probably just use the master version of the repo for the libraries.

File: ci/Jenkinsfile [hunk 2]
```diff
@@ -39,27 +25,18 @@ library identifier: "ci-pipeline@${env.ghprbActualCommit}",
...
-                                string(defaultValue: '', description: 'HTTP dir', name: 'HTTP_DIR'),
-                                string(defaultValue: 'ci-pipeline', description: 'Main project repo', name: 'PROJECT_REPO'),
-                                string(defaultValue: '', description: 'Main topic to publish on', name: 'MAIN_TOPIC'),
-                                string(defaultValue: '', description: 'Principal for authenticating with fedora build system', name: 'FEDORA_PRINCIPAL'),
                                 string(defaultValue: 'master', description: '', name: 'ghprbActualCommit'),
                                 string(defaultValue: 'CentOS-PaaS-SIG/ci-pipeline', description: '', name: 'ghprbGhRepository'),
-                                string(defaultValue: '', description: '', name: 'sha1'),
-                                string(defaultValue: '', description: '', name: 'ghprbPullId'),
-                                string(defaultValue: '', description: '', name: 'ghprbPullAuthorLogin'),
                                 string(defaultValue: 'stable', description: 'Tag for rpmbuild image', name: 'SLAVE_TAG'),
```
Comment (johnbieren) [inline ci/Jenkinsfile]: nit - description for the container tags

File: ci/Jenkinsfile (hunk 1 above)
Comment (johnbieren) [reply ci/Jenkinsfile]: Here is an example of how I called the ci-pipeline libraries in a separate Jenkinsfile I created
https://github.com/johnbieren/systemd/blob/master/OpenShift/Jenkinsfile#L34

Comment (accorvin) [reply ci/Jenkinsfile]: 👍

File: ci/Jenkinsfile (hunk 2 above)
Comment (accorvin) [reply ci/Jenkinsfile]: Good catch. Thought I had fixed that.

**Comment by accorvin** [comment]
Closing this for now until I get a CI job working for this.

**Pull request closed without merging**